observer/probers/http/http_conf_test.go

```diff
@@ -84,25 +86,27 @@ func TestHTTPProberName(t *testing.T) {
 url: https://www.google.com
 rcodes: [ 200 ]
 useragent: ""
+insecure: true
 `
 	c := HTTPConf{}
 	configurer, err := c.UnmarshalSettings([]byte(proberYAML))
 	test.AssertNotError(t, err, "Got error for valid prober config")
 	prober, err := configurer.MakeProber(nil)
 	test.AssertNotError(t, err, "Got error for valid prober config")
-	test.AssertEquals(t, prober.Name(), "https://www.google.com-[200]-letsencrypt/boulder-observer-http-client")
+	test.AssertEquals(t, prober.Name(), "https://www.google.com-[200]-letsencrypt/boulder-observer-http-client-insecure")
 
 	// Test with custom `useragent`
 	proberYAML = `
 url: https://www.google.com
 rcodes: [ 200 ]
 useragent: fancy-custom-http-client
+insucre: false
```

Comment:
```suggestion
insecure: false
```

Or even better yet, remove this line entirely, to show that the default value works correctly.